spec/models/user_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
I would add something here for posterity... something like (may need to adjust a bit):

```  describe 'DB table' do
    it { is_expected.to have_db_column :email }
    it { is_expected.to have_db_column :password }
  end

  describe 'Validations' do
    it { is_expected.to validate_presence_of :email }
    it { is_expected.to validate_presence_of :encrypted_password }
  end

  describe 'Factory' do
    it 'should have valid Factory' do
      expect(FactoryBot.create(:user)).to be_valid 
    end
  end```

Reply:
Good idea. I'll add everything except the Factory. Factory is for a logged in user if I'm not mistaken?

Reply:
ahhhh alright. Will add it then